### PR TITLE
Skip installing thrust support in case nvcc not found in PATH.

### PR DESCRIPTION
**Pull request by takagi**
Currently, installing cupy fails with the following error message when nvcc is not found in PATH.

```
building 'cupy.cuda.thrust' extension
    error: unknown file type '.cu' (from 'cupy/cuda/cupy_thrust.cu')
```

This PR fix it to skip installing thrust support itself to avoid this inappropriate situation on environments where nvcc is not found.

### CODE REVIEW

**Comment by unnonouno** [comment]
Do we need to find `nvcc` command from `CUDA_PATH` or default CUDA directory in `/usr/local/cuda`? Is it dangerous?

**Comment by takagi** [comment]
Yes, we do. I missed what the installer script intends to find CUDA. The path of nvcc should be detected in the same way with looking up CUDA directory.

http://docs.cupy.chainer.org/en/latest/install.html#install-cupy-with-cuda

File: install/build.py [hunk 1]
```diff
@@ -15,23 +15,23 @@
 # provided functions are insufficient to implement cupy.linalg
 minimum_cusolver_cuda_version = 8000
 
-get_nvcc_path_warned = False
+_cuda_path = "NOT_INITIALIZED"
```
Comment (unnonouno) [inline install/build.py]: Use single quotes.

File: install/build.py [hunk 2]
```diff
@@ -15,23 +15,23 @@
...
-        utils.print_warning('nvcc not in path.',
-                            'Please set path to nvcc.')
-        get_nvcc_path_warned = True
-    return nvcc_path
+def get_cuda_path():
+    global _cuda_path
 
+    # Use a magic word to represent the cache not filled because None is a
+    # valid return value.
+    if _cuda_path is not "NOT_INITIALIZED":
```
Comment (unnonouno) [inline install/build.py]: ditto

File: install/build.py (hunk 1 above)
Comment (takagi) [inline install/build.py]: I have fixed.

File: install/build.py (hunk 2 above)
Comment (takagi) [inline install/build.py]: I have fixed.

**Comment by unnonouno** [comment]
LGTM

**Pull request merged**